app/javascript/components/App.js

```diff
@@ -5,13 +5,21 @@ import {
   Switch
 } from 'react-router-dom'
 
+import Navigation from './components/Navigation'
 import Home from './pages/Home'
+import NotFound from './components/NotFound'
 
 class App extends React.Component {
   render () {
     return (
       <>
-            <Home />
+        <Router>
+          <Navigation />
+          <Switch>
+            <Route exact path="/" component={Home} />
+            <Route component={NotFound}/>
+          </Switch>
+        </Router>
```

Comment:
Router should be the outermost tag

Reply:
The Router tag is enclosing Navigation and Switch (it's the first tag after the fragment and the last tag before the closing fragment)

Can you please advise what outermost is?